### DISCUSSION TITLE
Hi all,

### DISCUSSION THREAD

**Discussion by KonMost** [fabricjs/fabric.js#7687] [Q&A]
Hi all,

can someone help me on how to modify the circle_brush.class in order to get an ellipse/slanted line instead of a circle as brush tip to draw the freedrawing lines? i think this would be the place to add a calligraphy kind of brush. i tried changing my min.js but it did not take the slanted line or oval as brush tip. please advise

_Originally posted by @KonMost in https://github.com/fabricjs/fabric.js/issues/6719#issuecomment-1040317504_

**Comment by ShaMan123** [comment]
http://fabricjs.com/freedrawing

**Comment by ShaMan123** [comment]
it's best to provide a working example 
make it easier for others to help you

**Comment by ShaMan123** [comment]
duplicate. continue in the first
